src/command_modules/azure-cli-datalake/azure/cli/command_modules/datalake/store/custom.py

```diff
@@ -0,0 +1,257 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import codecs
+import os
+import re
+import time
+import uuid
+
+from OpenSSL import crypto
+
+from msrestazure.azure_exceptions import CloudError
+from azure.mgmt.datalake.store.models import (DataLakeStoreAccountUpdateParameters,
+                                        FirewallRule,
+                                        DataLakeStoreAccount,
+                                        EncryptionConfigType,
+                                        EncryptionIdentity,
+                                        EncryptionConfig,
+                                        EncryptionState,
+                                        KeyVaultMetaInfo)
+
+from azure.datalake.store.multithread import (ADLUploader, ADLDownloader)
+from azure.cli.command_modules.datalake.store._client_factory import (cf_datalake_store_filesystem)
+
+import azure.cli.core.telemetry as telemetry
+from azure.cli.core._util import CLIError
+import azure.cli.core.azlogging as azlogging
+
+logger = azlogging.get_az_logger(__name__)
+# account customiaztions
+def list_adls_account(client, resource_group_name=None):
+    account_list = client.list_by_resource_group(resource_group_name=resource_group_name) \
+        if resource_group_name else client.list()
+    return list(account_list)
+
+def create_adls_account(client, 
+                        resource_group_name,
+                        account_name,
+                        location,
+                        default_group = None,
+                        tags = None,
+                        encryption_type = None,
+                        key_vault_id = None,
+                        key_name = None,
+                        key_version = None,
+                        disable_encryption = False,
+                        tier = None):
+    create_params = DataLakeStoreAccount(location)
+    if tags:
+        create_params.tags = tags
+
+    if default_group:
+        create_params.default_group = default_group
+
+    if tier:
+        create_params.new_tier = tier
+    if not disable_encryption:
+        identity = EncryptionIdentity()
+        config = EncryptionConfig()
+        if not encryption_type:
+            logger.warning('No Encryption type passed in, defaulting to Service managed encryption. To opt out, explicitly pass in --disable_encryption')
+            encryption_type = EncryptionConfigType.service_managed
```

Comment:
You can set `EncryptionConfigType.service_managed.value` as the default value in the signature and it will be shown in the help text.  Also, it is highly atypical to display logger warnings in the CLI for using the default values.